core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.directive.html

```diff
@@ -245,4 +260,13 @@
   .oppia-audio-translation-bar {
     height: 50px;
   }
+  .oppia-rte-editor-focused:focus > * {
+    outline: solid 2px #009688;
+  }
+  .oppia-rte-editor-focused:focus > * > * {
+    outline: solid 2px #009688;
+  }
+  .oppia-rte-editor-focused *:hover {
+    background-color: rgba(233, 241, 242);
+  }
```

Comment:
Hm, I'm not sure about the use of wildcard selector (*) here. Should we be more specific with the CSS selectors? ie. specify the css class or element that you want the style to apply to.
/cc @DubeySandeep

Reply:
I think it is bad practice to use (*) because it becomes difficult to add rules to individual elements in the future and that makes it hard to maintain.

Reply:
Thanks for the suggestion, adding more specific selectors solved some issues I was having with some elements having unwanted outlines!